cmd/karavictl/cmd/storage_create.go

```diff
@@ -304,6 +307,34 @@ func NewStorageCreateCmd() *cobra.Command {
 					createStorageFunc(storageID)
 				}
 
+			case powerscale:
+				tempStorage = storage[powerscale]
+				if tempStorage == nil {
+					tempStorage = make(map[string]System)
+				}
+
+				psClient, err := pscale.NewClientWithArgs(context.Background(), epURL.String(), input.Insecure, 1, input.User, "Administrators", input.Password, "")
```

Comment:
Should "Administrators" be an input parameter instead of being hardcoded?

Reply:
`Administrators` is a default, un-deletable group on the PowerScale. AFAIK the only account that should be used `karavictl storage add` is an account that has administrator privileges on the PowerScale.

@alexemc do you think that hardcoding the `administrator` group for the PowerScale client is appropriate here?